### PR TITLE
New import export json action test

### PR DESCRIPTION
**Pull request by kobe1104**
### Resolves:
resolves: #4558 

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?

### Author Checklist

* [x] Changes address original issue?
* [x] Unit tests included and/or updated with changes?
* [x] Command line build passes?
* [x] Has this been smoke tested?
* [x] Testing instructions included in associated issue?

### CODE REVIEW

**Comment by nikhilmandlik** [comment]
@kobe1104  some conflicts

File: src/plugins/exportAsJSONAction/ExportAsJSONAction.js [hunk 1]
```diff
@@ -68,6 +68,10 @@ export default class ExportAsJSONAction {
         this._write(this.root);
     }
 
+<<<<<<< HEAD
```
Comment (nikhilmandlik) [inline src/plugins/exportAsJSONAction/ExportAsJSONAction.js:71]: merge issue?

**Review by davetsay — CHANGES_REQUESTED** [review]
Half of the tests are arrow functions and half are regular ole functions. Can we pick one for all?

A comment, not a request. I'm not sure I understand what the difference between these tests are, `ExportAsJSONAction exports object from tree`, `can export self-containing objects`.

Some inline comments. Looking good.

File: src/plugins/exportAsJSONAction/ExportAsJSONActionSpec.js [hunk 2]
```diff
@@ -0,0 +1,310 @@
+import ExportAsJSONActionPlugin from './plugin';
```
[the remark this answers is not in the document]
Comment (nikhilmandlik) [reply src/plugins/exportAsJSONAction/ExportAsJSONActionSpec.js:1]: good catch

**Comment by nikhilmandlik** [comment]
> Half of the tests are arrow functions and half are regular ole functions. Can we pick one for all?
> 
> A comment, not a request. I'm not sure I understand what the difference between these tests are, `ExportAsJSONAction exports object from tree`, `can export self-containing objects`.
> 
> Some inline comments. Looking good.

yeah updated, `can export self-containing objects` has infinite Parent and child relation, parent contains child and child contains parent. 
eg:
Screen Shot 2021-12-23 at 4 54 05 PM

**Review by davetsay — APPROVED** [review]
LGTM

## Reviewer Checklist
| | |
| --- | :---: |
| Changes appear to address issue? | Y |
| Appropriate unit tests included? | Y |
| Code style and in-line documentation are appropriate? | Y |
| Commit messages meet standards? | Y |
| Has associated issue been labelled `unverified`? | Y |

**Pull request merged**